Makefile

```diff
@@ -10,3 +10,7 @@ endif
 coveralls:
 	roveralls
 	goveralls -coverprofile=roveralls.coverprofile -service=travis-ci
+
+test: check
+
+build: go build -o /bin/chaoskube -v
```

Comment:
Do you intentionally use the global `/bin` instead of a local (relative) path?

Reply:
Good point, I can change it to be relative to the Makefile

Reply:
Let's do that. Otherwise it might have unintended effects.

Reply:
I've changed it to generate the build in a bin folder located in the project root
Can I add it to .gitignore so that local builds are not picked up by git?

Reply:
```
$ ~/chaoskube tree -L 2          
.
├── CHANGELOG.md
├── Dockerfile
├── Dockerfile.arm32v6
├── Dockerfile.arm64v8
├── Dockerfile.ppc64le
├── LICENSE
├── Makefile
├── README.md
├── bin
│   └── chaoskube    <--- here
├── chaoskube
│   ├── chaoskube.go
│   └── chaoskube_test.go
├── chaoskube.png
├── examples
│   ├── chaoskube.yaml
│   └── rbac.yaml
├── go.mod
├── go.sum
├── internal
│   └── testutil
├── main.go
├── metrics
│   └── metrics.go
├── notifier
│   ├── noop.go
│   ├── notifier.go
│   ├── notifier_test.go
│   ├── slack.go
│   └── slack_test.go
├── renovate.json
├── terminator
│   ├── delete_pod.go
│   ├── delete_pod_test.go
│   └── terminator.go
└── util
    ├── util.go
    └── util_test.go

9 directories, 29 files

```